CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+This document explains how to build the Wasm module of the Internet Identity canister from scratch.
+
+> 💡 Binary releases are available on the [release page][releases].
+
+> 💡 The simplest way to build the code is to use the [Docker build][docker-build].
+
+> 💡 Make sure to read up on the [build features and flavors][features-and-flavors].
+
+The build requires the following dependencies:
+
+* `dfx` version 0.8.3
+* [`ic-cdk-optimizer`](https://github.com/dfinity/cdk-rs/tree/main/src/ic-cdk-optimizer) version 0.3.1
+* Rust version 1.58.1
+* Node.js v16+
+* CMake
```

Comment:
Where do we have a dependency on that?

Reply:
I guess we don't! I just copy pasted that, removing.

Reply:
Actually I think it might be needed for `ic-cdk-optimizer`; will leave that in for now, but we'll probably remove the `ic-cdk-optimizer` dependency as well soon